.gitignore

```diff
@@ -0,0 +1,2 @@
+# Haskell stack build artifacts
+.stack-work/
```

Comment:
This probably shouldn't be in the repository.

Reply:
Done.